backend/src/common/enums/case/case.enum.ts

```diff
@@ -0,0 +1,5 @@
+enum Case {
+  KEBAB = 'kebab',
+}
```

Comment:
Maybe it would be better to name enum `StringCase`, or something like that - to provide more clear desc of its usage,
And still sure that it should be `snake_case` for `key column` values